tests/test_dsls.py

```diff
@@ -0,0 +1,127 @@
+"""This module tests the behavior of using excludes in elasticsearch mappings"""
```

Comment:
Should this also contain test cases for gene_centric to verify case_id is being returned?

Reply:
The test cases were testing the elasticsearch excludes behavior when used with wildcards. It was trying to answer the question, when I exclude using a wildcard of this form `case.*_*` does it behave as expected. I wasn't testing whether gene_id shows up in case centric or not (at least not directly). The specific test for whether the fields show up or not was done manually by building an index and checking

Reply:
Sounds good, thank you!